README.md

```diff
@@ -135,7 +135,7 @@ await yourToken.transfer(
 
 `contract Vendor is Ownable {`
 
-⚠️ You will also need to import the Ownable.sol contract!
+⚠️ You will also need to uncomment the Ownable.sol contract!
```

Comment:
Basically it's uncommenting of import, so probably `You will also need to uncomment the import of Ownable.sol contract!` is bettter?